edge-agent/docker/linux/amd64/Dockerfile

```diff
@@ -8,13 +8,6 @@ ARG EXE_DIR=.
 RUN apk update && \
     apk add --no-cache snappy
 
-# create a default user to allow agent to run as non-root
-# value was chosen as a large value to avoid a typical regular uid, 
-# and matches the "edgeagentuser" created in install packages.
-ARG EDGEAGENTUSER_ID
-ENV EDGEAGENTUSER_ID ${EDGEAGENTUSER_ID:-13622}
-RUN adduser -Ds /bin/sh -u ${EDGEAGENTUSER_ID} edgeagentuser 
-
```

Comment:
Why are you removing it from here? Because then the user is not added on host?

Reply:
Where does the user get created then?

Reply:
I see you moved it to the start script..

Reply:
I added it to the install script. The start scripts also do this; there's no need to also create the user here.